cmd/history/history.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Extract git history from the main kata repos and generate two report files:
+#  report.txt - commits merged, sorted by repo section, and date.
+#  sorted_report.txt - all commits sorted by date
+#
+# This information can be useful when tracking down 'what changed when'.
+
+set -e
+
+REPORTFILE="report.txt"
+
+# Set a date to start from, or use the default of '1 week'
+#START="2019-03-08"
```

Comment:
Might be worth making these CLI args/options so you could do:

```sh
$ history.sh
$ history.sh [-h|--help]
$ history.sh [start]
$ history.sh [start] [end]

```

Reply:
I guess... I could make this a full blow tool with help and allll. I guess.... more typing....

Reply:
Whilst you're at it, could you also add i18n support... oh and colour? And sound. Also, how about a gui?

/me runs...

Reply:
... and a `--display-ascii-unicorns` option just for fun :unicorn: